stetho/src/main/java/com/facebook/stetho/inspector/elements/DescriptorMap.java

```diff
@@ -0,0 +1,84 @@
+// Copyright 2004-present Facebook. All Rights Reserved.
+
+package com.facebook.stetho.inspector.elements;
+
+import com.facebook.stetho.common.Util;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public final class DescriptorMap {
+  private final Map<Class<?>, Descriptor> mMap = new HashMap<Class<?>, Descriptor>(); // TODO: ArrayMap
```

Comment:
This isn't a TODO really.  We shouldn't use the support library so it's not really an option.